<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientView</class>
 <widget class="QWidget" name="PatientView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>患者管理</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="textSearch">
       <property name="placeholderText">
        <string>请输入姓名进行查找</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btSearch">
       <property name="text">
        <string>查找</string>
       </property>
       <property name="icon">
        <iconset resource="lab4.qrc">
         <normaloff>:/img/search.png</normaloff>:/img/search.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btAdd">
       <property name="text">
        <string>添加</string>
       </property>
       <property name="icon">
        <iconset resource="lab4.qrc">
         <normaloff>:/img/add.png</normaloff>:/img/add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btDelete">
       <property name="text">
        <string>删除</string>
       </property>
       <property name="icon">
        <iconset resource="lab4.qrc">
         <normaloff>:/img/delete.png</normaloff>:/img/delete.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btEdit">
       <property name="text">
        <string>修改</string>
       </property>
       <property name="icon">
        <iconset resource="lab4.qrc">
         <normaloff>:/img/edit.png</normaloff>:/img/edit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="sexBox">
         <property name="title">
          <string>性别</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QRadioButton" name="radioMan">
                <property name="text">
                 <string>男</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioWoman">
                <property name="text">
                 <string>女</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QRadioButton" name="radioNormal">
              <property name="text">
               <string>不限</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ageBox">
         <item>
          <property name="text">
           <string>年龄顺序</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>年龄逆序</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="createdTimeBox">
         <item>
          <property name="text">
           <string>创建时间顺序</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>创建时间逆序</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="btimport">
           <property name="text">
            <string>导入</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btExport">
           <property name="text">
            <string>导出</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btPrev">
       <property name="text">
        <string>上一页</string>
       </property>
       <property name="icon">
        <iconset resource="lab4.qrc">
         <normaloff>:/img/prev.png</normaloff>:/img/prev.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="totalAndCurrent">
       <property name="text">
        <string>第Y页</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btNext">
       <property name="text">
        <string>下一页</string>
       </property>
       <property name="icon">
        <iconset resource="lab4.qrc">
         <normaloff>:/img/next.png</normaloff>:/img/next.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="lab4.qrc"/>
 </resources>
 <connections/>
</ui>
